builder/utils.go

```diff
@@ -0,0 +1,34 @@
+package builder
```

Comment:
any reason why this is called `utils.go` vs `ssz.go` ?

Reply:
that makes sense - I don't feel too strongly about this but I think it would be nice if we determine another name for such a file. `utils` is an anti pattern in Go. Many projects still use it but we should try to stick to idioms when they make sense.

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common